cmake/deps-utils/CoCoALib-0.99800-trace.patch

```diff
@@ -1,3 +1,16 @@
+diff --git a/configuration/shell-fns.sh b/configuration/shell-fns.sh
+index 190dbb4..c8d281f 100755
+--- a/configuration/shell-fns.sh
++++ b/configuration/shell-fns.sh
+@@ -11,7 +11,7 @@ mktempdir()
+ {
+     TODAY=`date "+%Y%m%d"`
+     TIME=`date "+%H%M%S"`
+-    TMP_DIR="/tmp/CoCoALib-config/$USER-$TODAY/$1-$TIME-$$"
++    TMP_DIR="/tmp/CoCoALib-config-$USER/$USER-$TODAY/$1-$TIME-$$"
```

Comment:
Why is this change necessary?

Reply:
@Sarkoxed : I'm curious about this too.

Reply:
Hey @aniemetz, @alex-ozdemir! Sorry for this huge delay.

Regarding the change in the cocoalib patch: I struggled to create a cvc5 for each user on a public server, since the directory `/tmp/CoCoALib-config` has write permissions only for the first user. So, with this patch, it will create a directory for each user. I can remove it from this PR if you don't want this behavior.